doc/gpo/PowerToys.adml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (c) Microsoft Corporation.
+     Licensed under the MIT License. -->
+<policyDefinitionResources xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" revision="1.0" schemaVersion="1.0" xmlns="http://schemas.microsoft.com/GroupPolicy/2006/07/PolicyDefinitions">
+  <displayName>PowerToys</displayName>
+  <description>PowerToys</description>
+  <resources>
+    <stringTable>
+      <string id="PowerToys">Microsoft PowerToys</string>
+      <string id="ConfigureEnabledUtilityDescription">This policy configures the enabled state for a PowerToys utility.
+
+If you enable this setting, the utility will be always enabled and the user won't be able to disable it.
+
+If you disable this setting, the utility will be always disabled and the user won't be able to enable it.
+
+If you don't configure this setting, users are able to disable or enable the utility.
+</string>
+      <string id="ConfigureEnabledUtilityAlwaysOnTop">Always On Top: Configure enabled state</string>
+      <string id="ConfigureEnabledUtilityAwake">Awake: Configure enabled state</string>
+      <string id="ConfigureEnabledUtilityColorPicker">Color Picker: Configure enabled state</string>
+      <string id="ConfigureEnabledUtilityFancyZones">FancyZones: Configure enabled state</string>
+      <string id="ConfigureEnabledUtilityFileExplorerSVGPreview">SVG file preview: Configure enabled state</string>
+      <string id="ConfigureEnabledUtilityFileExplorerMarkdownPreview">Markdown file preview: Configure enabled state</string>
+      <string id="ConfigureEnabledUtilityFileExplorerMonacoPreview">Source code file preview: Configure enabled state</string>
+      <string id="ConfigureEnabledUtilityFileExplorerPDFPreview">PDF file preview: Configure enabled state</string>
+      <string id="ConfigureEnabledUtilityFileExplorerGcodePreview">Gcode file preview: Configure enabled state</string>
+      <string id="ConfigureEnabledUtilityFileExplorerSVGThumbnails">SVG file thumbnail: Configure enabled state</string>
+      <string id="ConfigureEnabledUtilityFileExplorerPDFThumbnails">PDF file thumbnail: Configure enabled state</string>
```

Comment:
We should add a warning to the description like we have in the ux.

Reply:
I think thumbnails don't conflict.